src/live_restore/live_restore_fs.c

```diff
@@ -1295,6 +1370,16 @@ __live_restore_fs_open_file(WT_FILE_SYSTEM *fs, WT_SESSION *wt_session, const ch
     lr_fh->iface.fh_extend = NULL;
     lr_fh->iface.fh_extend_nolock = NULL;
 
+    WT_ERR(__wt_rwlock_init(session, &lr_fh->ext_lock));
+
+    /* FIXME-WT-13823 Handle the exclusive flag and other flags */
+
+    if (file_type == WT_FS_OPEN_FILE_TYPE_DIRECTORY) {
```

Comment:
Nit: No brackets for single line ifs.

Reply:
[1fe24c5](https://github.com/wiredtiger/wiredtiger/pull/11379/commits/1fe24c5407d375cef1de5a0536e4ee4aa57eee51)